mobly/controllers/android_device_lib/jsonrpc_client_base.py

```diff
@@ -297,14 +298,14 @@ def _client_receive(self):
         try:
             response = self._client.readline()
             response_log = str(response)
-            if self.verbose_logging or\
-                _MAX_RPC_RETURN_LENGTH_IN_DEBUG_LOG >= len(response):
+            if self.verbose_logging or _MAX_RPC_RESP_LOGGING_LENGTH >= len(
```

Comment:
I don't see why we need the `or  _MAX_RPC_RESP_LOGGING_LENGTH...` part.
if verbose is on, we simply log everything right?

Reply:
If verbose is on, we simply log everything, that's right.

And if the verbose is off, but the rpc return less then 1024 chars, we also log everything, that's the `or _MAX_RPC_RESP_LOGGING_LENGTH...` part do

Reply:
You should explicitly check for this only when verbose logging is off so the logic is more clear?

Reply:
Or just not check it at all?